CMakeLists.txt

```diff
@@ -5,10 +5,10 @@ cmake_minimum_required( VERSION 3.18 )
 # see also -DADD_ below
 option( USE_FORTRAN "Fortran is required for some tester checks, but can be disabled with reduced functionality" ON )
 
-if (USE_FORTRAN)
-    project( MAGMA C CXX Fortran )
+if(USE_FORTRAN)
+  project( MAGMA LANGUAGES C CXX Fortran )
 else()
-    project( MAGMA C CXX )
+  project( MAGMA LANGUAGES C CXX )
```

Comment:
Please don't change the formatting. We use 4-space indents, and space after `if`, the same as C++ code. (Although our CMake file is not entirely consistent with that and needs to be updated to 4-space indents.) The `LANGUAGES` keyword is optional, [per CMake's docs](https://cmake.org/cmake/help/latest/command/project.html#project), so just leave well enough alone.